### ISSUE TITLE
Execute on shell to recoginze

### ISSUE THREAD

**Issue by haxpor** [c-w/ghp-import#63]
I tried executing `ghp-import` and it works great without `-s` command.

The command I executed is like this 

```
ghp-import -c 'mysubdomain.domain.com' -m 'Deployed via ghp-import' -p -b 'gh-pages' ./docs/
```

But because I've set up git to [sign commit](https://help.github.com/articles/signing-commits-using-gpg/) + bunch of step to make it automatically sign every time you commit via `git commit` so you don't have to enter password every time. These workflow is not recognized anymore if execute with command above, ...

thus I tried with `-s` option as I think it will take all setup in `~/.bash_profile` and shell environment into effect but with no luck. I always get the following error

```
ghp-import -c 'mysubdomain.domain.com' -m 'Deployed via ghp-import' -p -b 'gh-pages' -s ./docs/
Usage: ghp-import [OPTIONS] DIRECTORY

ghp-import: error: Unknown Git error
```

> The command executed is same as the first one except that it has `-s` at the end before directory parameter.

I could verify whether it's successful or not, by taking a look at commit in which it should have `verified` tagged along side on Github website. The command without `-s` successfully committed but with no `verified` tag. On side note, `mkdocs` also behaves the same with no `verfiied` tag. Any idea to solve this issue?

**Comment by danielhoherd** [comment]
I was about to file a bug and was issuing `-s` to find additional information before submitting when I encountered this same error.

- macOS 10.13.2
- Python 3.6.4 (from homebrew)
- ghp-import==0.5.5 (from pip3)

**Comment by c-w** [comment]
@danielhoherd @haxpor I just merged a pull request that I believe should fix this issue. Please let me know if this is still a problem for you.

**Comment by c-w** [comment]
This has been released in [0.6.0](https://pypi.org/project/ghp-import/0.6.0/).
